composer.json

```diff
@@ -52,7 +52,6 @@
                 "Allow class property types instead of @var comments": "https://www.drupal.org/files/issues/2021-09-21/coder-3123282-3.patch"
             },
             "drupal/core": {
-                "Color module html preview optional": "https://www.drupal.org/files/issues/color-optional-html-preview-2844190-2.patch",
```

Comment:
@ronaldtebrake Would it possible for you to take a look at this one? 🙈

Reply:
It's from 2017, I bet I know as much as you do there haha! :) Plus, you might be able to get some drupal core commit credits ;)

Reply:
@ronaldtebrake smart ! ;-)

Reply:
I think we don't need this anymore.